src/Raven.Server/Raven.Server.csproj

```diff
@@ -144,6 +144,9 @@
     <PackageReference Include="NuGet.Protocol" Version="5.8.1" />
     <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="2.19.101" />
     <PackageReference Include="Portable.BouncyCastle" Version="1.8.9" />
+    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="2.19.100" />
```

Comment:
We already have it referenced - see above:

`<PackageReference Include="Oracle.ManagedDataAccess.Core" Version="2.19.101" />`